### PR TITLE
[one-cmds] Tests for one-quantize

### PR DESCRIPTION
**Pull request by seanshpark**
This will introduce test scripts for one-quantize

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #4326, from draft #4440, test case from https://github.com/Samsung/ONE/issues/4155#issuecomment-690903095

File: compiler/one-cmds/tests/one-quantize_001.test [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+trap trap_err_onexit ERR
+
+inputfile="./inception_v3.circle"
+outputfile="./inception_v3.quantized.circle"
+
+rm -rf ${outputfile}
+
+# to create inception_v3.circle
+if [[ ! -s ${inputfile} ]]; then
+  /bin/bash one-import_001.test >> /dev/null
```
Comment (mhs4670go) [inline compiler/one-cmds/tests/one-quantize_001.test:35]: Maybe  `one-import_001.test` will be merged later?

Comment (seanshpark) [reply compiler/one-cmds/tests/one-quantize_001.test:35]: `one-import_001.test` is in another PR. Test is not enabled yet so it is OK.

**Comment by mhs4670go** [comment]
Except import test, they tests the process that follow import. So, `one-import_001` can be run in the prepare section first, which reduces test time I think.

**Comment by seanshpark** [comment]
> So, one-import_001 can be run in the prepare section first

I'll post another PR for `prepare_test_materials.sh` file patch

> which reduces test time I think.

I've updated with `one-import_001.test` PR so that `circle` file is not deleted.
Current code to call to `one-import_001.test` would be good to stay as-is for safety and will be called when file doesn't exist.

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**